src/widget/OktaSignIn.ts

```diff
@@ -42,6 +42,10 @@ export default class OktaSignIn implements OktaSignInAPI {
       `);
 
     this.options = options;
+    if (window?.oktaData?.signIn?.stateToken) {
+      // Okta hosted custom SIW
+      this.options.stateToken = window?.oktaData?.signIn?.stateToken;
```

Comment:
I am also contemplating setting the stateToken in the Admin sign in page code editor:
```
config.stateToken = oktaData.signIn.stateToken; // required
```